DEFI/ERC20/certora/specs/ERC20Full.spec

```diff
@@ -0,0 +1,736 @@
+methods {
```

Comment:
header, someting like:

/***
This example is a full spec for erc20. 
to run this...
example of a run: link to run
mutation test for this spec:  .... 
See https://docs.certora.com for a complete guide.
***/

/*
    Declaration of methods that are used in the rules. `envfree` indicates that
    the method is not dependent on the environment (`msg.value`, `msg.sender`).
    Methods that are not declared here are assumed to be dependent on the
    environment.
*/
and give credit to openzepplin spec if we took from there

Reply:
Done